src/main/java/appeng/parts/automation/PartAnnihilationPlane.java

```diff
@@ -359,7 +359,36 @@ private boolean handleOverflow( EntityItem entityItem, IAEItemStack overflow )
 		entityItem.getEntityItem().stackSize = newStackSize;
 
 		return changed;
+	}
+
+	/**
+	 * Spawns an overflow item as new {@link EntityItem} into the {@link World}
+	 *
+	 * @param w the world to spawn it
+	 * @param x x coordinate
+	 * @param y y coordinate
+	 * @param z coordinate
+	 * @param overflow the item to spawn
+	 */
+	private void spawnOverflow( IAEItemStack overflow )
+	{
+		if( overflow == null )
+		{
+			return;
+		}
```

Comment:
not sure if this method is the place where to silently reject no overflow, should probably be checked in the loop